source/settings/enable-dev-tools.txt

```diff
@@ -0,0 +1,106 @@
+.. _compass-enable-dev-tools:
+
+======================
+Toggle Chrome DevTools
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can toggle the ``enableDevTools`` option to control user access to Chrome 
+DevTools on |compass|. 
+
+About This Task
+---------------
+
+By default, :guilabel:`Enable DevTools` is enabled.
+
+If you set :guilabel:`Set Read-Only Mode`, :guilabel:`Enable MongoDB Shell`, 
+:guilabel:`maxTimeMS`, or :guilabel:`Protect Connection String Secrets`, 
+|compass-short| automatically disables the :guilabel:`Enable DevTools` setting.
+
+Procedure
+---------
+
+You can set the ``enableDevTools`` option in either: 
+
+- The |compass-short| :ref:`Settings panel <compass-settings-reference>`
+
+- The :ref:`command line <cli-options>`
+
+- A :ref:`configuration file <config-file>`
+
+|compass-short| Settings Panel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: In the |compass-short| top menu bar, click :guilabel:`MongoDB Compass`. 
+
+      .. figure:: /images/compass/settings-panel-location.png
+         :scale: 60% 
+         :alt: Settings panel location under the MongoDB Compass system menu
+
+      Alternatively, you can use keyboard shortcuts to open the 
+      :guilabel:`Settings` panel: 
+
+      - Windows / Linux: ``Ctrl`` + ``,``
+
+      - Mac: ``⌘`` + ``,``
+
+   .. step:: In the :guilabel:`MongoDB Compass` menu, click :guilabel:`Settings`.
+
+      |compass-short| opens a dialog box where you can configure your |compass| 
+      settings.
+
+   .. step:: Toggle :guilabel:`Enable DevTools`.
+
+   .. step:: Click :guilabel:`Save`.
+
+Command Line Example
```

Comment:
Ah I mean for lines 31-37 to act as that sort of blurb that explains the different ways you can set the devtools option, and the "Settings Panel", "Command Line Example", and "Configuration File Example" headers are the different ways you can follow the procedure. 

I'll probably take out "example" from these headers since that can cause them to be confused with examples instead of procedures.